ister.py

```diff
@@ -1243,6 +1247,9 @@ def handle_options():
     parser.add_argument("-k", "--kcmdline", action="store",
                         default="/proc/cmdline",
                         help="File to inspect for kernel cmdline opts")
+    parser.add_argument("-S", "--state", action="store",
```

Comment:
Also, instead of state being the argument statedir would be a more fitting name.